js/web/script/build.ts

```diff
@@ -41,28 +44,56 @@ function validateFile(path: string): void {
   }
 }
 
+npmlog.info('Build.Bundle', 'Retrieving npm bin folder...');
+const npmBin = execSync('npm bin', {encoding: 'utf8'}).trimRight();
+npmlog.info('Build.Bundle', `Retrieving npm bin folder... DONE, folder: ${npmBin}`);
+
 if (WASM) {
   npmlog.info('Build', 'Validating WebAssembly artifacts...');
   try {
-    validateFile(WASM_JS_PATH);
-    validateFile(WASM_THREADED_JS_PATH);
+    validateFile(WASM_BINDING_JS_PATH);
+    validateFile(WASM_BINDING_THREADED_JS_PATH);
+    validateFile(WASM_BINDING_THREADED_WORKER_JS_PATH);
     validateFile(WASM_WASM_PATH);
     validateFile(WASM_THREADED_WASM_PATH);
-    validateFile(WASM_THREADED_WORKER_JS_PATH);
   } catch (e) {
     npmlog.error('Build', `WebAssembly files are not ready. build WASM first. ERR: ${e}`);
     throw e;
   }
   npmlog.info('Build', 'Validating WebAssembly artifacts... DONE');
+
+  npmlog.info('Build', `Copying file "ort-wasm-threaded.js" to "${WASM_DIST_FOLDER}"...`);
+  try {
+    fs.copyFileSync(WASM_BINDING_THREADED_JS_PATH, WASM_THREADED_JS_PATH);
+    validateFile(WASM_THREADED_JS_PATH);
+  } catch (e) {
+    npmlog.error('Build', `Failed to copy file. ERR: ${e}`);
+    throw e;
+  }
+  npmlog.info('Build', `Copying file "ort-wasm-threaded.js" to "${WASM_DIST_FOLDER}"... DONE`);
+
+  npmlog.info('Build', `Copying file "ort-wasm-threaded.worker.js" to "${WASM_DIST_FOLDER}"...`);
+  try {
+    fs.copyFileSync(WASM_BINDING_THREADED_WORKER_JS_PATH, WASM_THREADED_WORKER_JS_PATH);
+    validateFile(WASM_THREADED_WORKER_JS_PATH);
+  } catch (e) {
+    npmlog.error('Build', `Failed to copy file. ERR: ${e}`);
+    throw e;
+  }
+  npmlog.info('Build', `Copying file "ort-wasm-threaded.worker.js" to "${WASM_DIST_FOLDER}"... DONE`);
+
+  npmlog.info('Build', 'Minimizing generated JavaScript files...');
+  // const terserCommand = path.join(npmBin, 'terser');
```

Comment:
Uncomment these ones? We're going to put a license header to all js files.

Reply:
file `ort-wasm.js`, `ort-wasm-threaded.js` and `ort-wasm-threaded.worker.js` are generated by emscripten. the first file is bundled inside `ort.min.js`; and the later 2 files are both embeded inside `ort.min.js` and also put in folder `/js/web/dist/` as well (this is for use in Node.js). And yes we need special step to add license header to these 2 files in folder `/js/web/dist/`.

Reply:
license header is added. now we have multiple instances of those files:
`/build/**/ort-wasm-threaded.js` <--- this is where emscripten output is
`/js/web/lib/wasm/binding/ort-wasm-threaded.js` <---- this is where we copy the file manually to, before building ort-web
`/js/web/lib/wasm/binding/ort-wasm-threaded.min.js` <---- this is where `build.ts` use terser to generate the minimized file
`/js/web/dist/ort-wasm-threaded.js` <--- this is copied from `/js/web/lib/wasm/binding/ort-wasm-threaded.min.js`, with copyright banner appended.